qutip/installsettings.py

```diff
@@ -115,5 +117,7 @@ class InstallSettings:
         # configured, rather than through _logging.get_logger().
         "_logger": _logger,
         # Running on mac with openblas make eigh unsafe
-        "eigh_unsafe": _eigh_unsafe
+        "eigh_unsafe": _eigh_unsafe,
+        # int size for the shape of data layer object
+        "idxint_size": None
```

Comment:
Could we set idxint_size here rather than in `__init__`?

Reply:
Sadly no, it create a circular import.
We may be able to set it in `qutip/core/data/base.pyx` instead of `__init__.py`.

Reply:
Oh well, too bad.